### PR TITLE
records: limit commas in name to two

### PR DESCRIPTION
**Pull request by miguelgrc**
* INSPIR-2974

### CODE REVIEW

File: tests/unit/test_author_builder.py [hunk 1]
```diff
@@ -917,10 +917,10 @@ def test_add_advisor_normalizes_name():
     subschema = schema['properties']['advisors']
 
     author = AuthorBuilder()
-    author.add_advisor('Riccardo Torres Jr')
+    author.add_advisor('Riccardo Torres')
 
     expected = [{
-        "name": 'Torres, Riccardo, Jr.',
```
Comment (michamos) [inline tests/unit/test_author_builder.py]: As you can tell by this test, the commit is wrong. In most fields, two commas should be allowed.

Comment (miguelgrc) [reply tests/unit/test_author_builder.py]: Then I think there's some issue in the [refinement](https://its.cern.ch/jira/browse/INSPIR-2974) @drjova

Comment (drjova) [reply tests/unit/test_author_builder.py]: @michamos it should be up to two commas per name?

Comment (michamos) [reply tests/unit/test_author_builder.py]: yes, precisely because of the suffixes like here.

Comment (drjova) [reply tests/unit/test_author_builder.py]: 👍 @miguelgrc my bad sorry

Comment (miguelgrc) [reply tests/unit/test_author_builder.py]: No worries @drjova, I'll change it :)

**Pull request merged**